platform/index.rst

```diff
@@ -8,3 +8,4 @@ Platform
   authorized_api/users/users_api
   authorized_api/apps/apps_api
   authorized_api/access_control/access_control_api
+  authorized_api/user_groups/user_groups_api
```

Comment:
Can we rearrange that index alphabetically? 

![obraz](https://user-images.githubusercontent.com/9349437/60869288-fab6f380-a22e-11e9-8b52-dc2c800d7000.png)

It's small improvement but would look a lot better imo. Maybe with an exception for `Authorized Api Guide` as it's an introduction how to connect with our api. I would leave it at top.